### PR TITLE
ES Update index.html

### PR DESCRIPTION
**Pull request by iammelea**

### CODE REVIEW

File: es/index.html [hunk 1]
```diff
@@ -192,95 +192,95 @@ <h2 class="display-4 mx-xl-6">Kusama?</h2>
                             </div>
                             <div class="collapse" id="panel-1">
                                 <div class="pt-3">
-                                    <p>Kusama is an early, unaudited and unrefined release of Polkadot. Kusama will serve as a proving ground, allowing teams and developers to build and deploy a parachain or try out Polkadot’s governance, staking, nomination and validation functionality in a real environment.</p>
+                                    <p>Kusama es un lanzamiento temprano, no auditado y sin refinar de Polkadot. Kusama servirá como campo de pruebas, permitiendo a los equipos y desarrolladores construir y desplegar una parachain o probar la funcionalidad de gobierno, delegaciones (Staking), nominación y validación de Polkadot en un entorno real.</p>
```
Comment (LaloGarza) [inline es/index.html]: It’s better to use 'anticipado' instead of 'temprano'

Comment (iammelea) [reply es/index.html]: done

File: es/index.html [hunk 2]
```diff
@@ -325,39 +325,39 @@ <h2 class="display-4 mx-xl-6">Kusama?</h2>
...
                                             <div class="custom-control custom-checkbox">
                                                 <input type="checkbox" class="custom-control-input" value="4" name="group[23917][4]" id="mce-group[23917]-23917-2">
-                                                <label class="custom-control-label text-small" for="mce-group[23917]-23917-2">I'm a validator</label>
+                                                <label class="custom-control-label text-small" for="mce-group[23917]-23917-2">Soy un validador</label>
                                             </div>
                                             <div class="custom-control custom-checkbox">
                                                 <input type="checkbox" class="custom-control-input"  value="8" name="group[23917][8]" id="mce-group[23917]-23917-3">
-                                                <label class="custom-control-label text-small" for="mce-group[23917]-23917-3">I'm into governance</label>
+                                                <label class="custom-control-label text-small" for="mce-group[23917]-23917-3">Estoy en el gobierno</label>
```
Comment (LaloGarza) [inline es/index.html]: Better to use 'Estoy en gobernanza'

Comment (iammelea) [reply es/index.html]: done

**Comment by iammelea** [comment]
> Please take a look at all the sentences when you are referring to a person in Spanish you are using mix text as if you’re talking as 'tu' & 'usted'. Check lines 225 & 239 as example: 225-'Puedes' & 239-'puede'.

suggestion= usted puede?

**Comment by LaloGarza** [comment]
> > Please take a look at all the sentences when you are referring to a person in Spanish you are using mix text as if you’re talking as 'tu' & 'usted'. Check lines 225 & 239 as example: 225-'Puedes' & 239-'puede'.
> 
> suggestion= usted puede?

Are you gonna be using terms as 'usted' or 'tu'?
'Puede' is for using terms as "usted"
'Puedes' is using terms as "tu"
This something that we use in Spanish, 'tu' for a more direct/fresh conversation or 'usted' for a more serious/respectful (like talking to an elderly person) conversation.

**Comment by iammelea** [comment]
better translate literal imo see now=
What can I do with KSM? = ¿Qué puedo hacer yo con KSM?
You can find out more =  Tú puedes encontrar más
 you can validate, nominate validators=  Tú puedes validar, nominar validadores, abonar parachains

ok?

**Comment by LaloGarza** [comment]
> better translate literal IMO see now=
> What can I do with KSM? = ¿Qué puedo hacer yo con KSM?
> You can find out more = Tú puedes encontrar más
> you can validate, nominate validators= Tú puedes validar, nominar validadores, abonar parachains
> 
> ok?

You don’t need to use 'tu' just use the right terms for that tone:
*¿Qué puedo hacer con KSM?
*Puedes encontrar más
*Puedes validar, nominar validadores, abonar parachains

**Pull request merged**